packages/config/config/devices/0x031e/vzw31-sn.json

```diff
@@ -0,0 +1,699 @@
+		},
+		{
+			"#": "94[0xff]",
+			"label": "LED7 Strip Effect - Duration",
+			"$import": "templates/inovelli_templates.json#led_effect_duration"
+		},
+		{
+			"#": "94[0xff00]",
+			"label": "LED7 Strip Effect - Level",
+			"$import": "templates/inovelli_templates.json#led_effect_level"
+		},
+		{
+			"#": "94[0xff0000]",
+			"label": "LED7 Strip Effect - Color",
+			"$import": "templates/inovelli_templates.json#led_effect_color"
+		},
+		{
+			"#": "94[0xff000000]",
+			"label": "LED7 Strip Effect - Effect",
+			"$import": "templates/inovelli_templates.json#led_effect_effect"
+		},
+		{
+			"#": "95",
+			"label": "Default All LED Strip Color When On",
+			"$import": "templates/inovelli_templates.json#hue_color_wheel_white",
+			"valueSize": 1,
+			"defaultValue": 170
+		},
+		{
+			"#": "96",
+			"label": "Default All LED Strip Color When Off",
+			"$import": "templates/inovelli_templates.json#hue_color_wheel_white",
+			"valueSize": 1,
+			"defaultValue": 170
+		},
+		{
+			"#": "97",
+			"label": "Default All LED Strip Brightness When On",
+			"$import": "~/templates/master_template.json#base_0-100_nounit",
+			"defaultValue": 33
+		},
+		{
+			"#": "98",
+			"label": "Default All LED Strip Brightness When Off",
+			"$import": "~/templates/master_template.json#base_0-100_nounit"
+		},
+		{
+			"#": "99[0xff]",
+			"label": "All LED Strip Effect - Duration",
+			"$import": "templates/inovelli_templates.json#led_effect_duration"
+		},
+		{
+			"#": "99[0xff00]",
+			"label": "All LED Strip Effect - Level",
+			"$import": "templates/inovelli_templates.json#led_effect_level"
+		},
+		{
+			"#": "99[0xff0000]",
+			"label": "All LED Strip Effect - Color",
+			"$import": "templates/inovelli_templates.json#led_effect_color"
+		},
+		{
+			"#": "99[0xff000000]",
+			"label": "All LED Strip Effect - Effect",
+			"$import": "templates/inovelli_templates.json#all_led_effect_effect"
+		},
+		{
+			"#": "100",
+			"label": "LED Brightness Scaling",
+			"description": "Set the LED lightbar scaling to match Inovelli's VZM31 (Blue Series) or LZW31 (Red Series) brightness.",
+			"valueSize": 1,
+			"minValue": 0,
+			"maxValue": 1,
+			"defaultValue": 0,
+			"unsigned": true,
+			"allowManualEntry": false,
+			"options": [
+				{
+					"label": "VZM31",
+					"value": 0
+				},
+				{
+					"label": "LZW31",
+					"value": 1
+				}
+			]
+		},
+		{
+			"#": "123",
+			"label": "Aux Switch Scenes",
+			"description": "Send different scene numbers when the Aux switch is clicked",
+			"$import": "~/templates/master_template.json#base_enable_disable"
+		},
+		{
+			"#": "158",
+			"label": "Dimmer Mode",
+			"valueSize": 1,
+			"minValue": 0,
+			"maxValue": 1,
+			"defaultValue": 1,
+			"allowManualEntry": false,
+			"options": [
+				{
+					"label": "Dimmer mode",
+					"value": 0
+				},
+				{
+					"label": "On/off mode",
+					"value": 1
+				}
+			]
+		},
+		{
+			"#": "159",
+			"label": "Only One LED Mode",
```

Comment:
Since this device can work as either an on/off switch or dimmer, this mode is to emulate the status LEDs on Inovelli's old on/off switches, which only had a single status LED.  So, if you want your new 2-in-1 switch to sort of look like your old on/off switches, you can turn on One LED Mode to not use all 7 of the LEDs in the status LED bar.

I can ask if anyone has a better idea for the name.

Reply:
Maybe just "Single Status LED" then?

Reply:
I've redone parameter 159 completely to match what's being used for Hubitat.  Hopefully that's clearer.

Reply:
@AlCalzone , is this change acceptable?

Reply:
👌🏻